stock_manual_shipping_weight/views/stock_picking_view.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <record id="view_stock_picking_form_inherit" model="ir.ui.view">
+            <field name="name">stock.picking.form.inherit</field>
+            <field name="model">stock.picking</field>
+            <field name="inherit_id" ref="stock.view_picking_form"/>
+            <field name="arch" type="xml">
+        		<xpath expr="//field[@name='shipping_weight']/.." position="inside">
```

Comment:
The opening and closing xpath tabs are wrongly tabulated.

Reply:
Done